src/bin/doc/package-lock.json

```diff
@@ -1,13 +0,0 @@
-{
-  "name": "node-doc-generator",
-  "version": "0.0.0",
-  "lockfileVersion": 1,
-  "requires": true,
-  "dependencies": {
-    "marked": {
-      "version": "1.1.1",
-      "resolved": "https://registry.npmjs.org/marked/-/marked-1.1.1.tgz",
-      "integrity": "sha512-mJzT8D2yPxoPh7h0UXkB+dBj4FykPJ2OIfxAWeIHrvoHDkFxukV/29QxoFQoPM6RLEwhIFdJpmKBlqVM3s2ZIw=="
-    }
-  }
-}
```

Comment:
Why was this file deleted?

Reply:
Weird that it removed it..  It should have re-added it when doing ``npm i``